### DISCUSSION TITLE
Can you port some node steam libs to rust?

### DISCUSSION THREAD

**Discussion by feeops** [dyc3/steamguard-cli#244] [General]
### Description

Some node steam lib are very popular.

For example

https://github.com/DoctorMcKay/node-steamcommunity
https://github.com/DoctorMcKay/node-steam-user
https://github.com/DoctorMcKay/node-steam-tradeoffer-manager

Can you develop rust version?

**Comment by dyc3** [comment]
I've considered it, but I've got other projects on my plate right now.
